### PR TITLE
Add MSA 2.0 details to Release Notes

### PR DESCRIPTION
**Pull request by thomasbroadwell**

### CODE REVIEW

File: Release Notes/2018/2018-05-07-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -1,16 +1,54 @@
...
+
+###### Service enablement
+Going forward customers can enable Managed Services by "Provider" instead of enabling by instance. Provider is a representation of an Account in the downstream cloud platform. Once Managed Services are enabled on a Provider, CenturyLink will provide full support on the services listed above.
+
+###### Billing
+Going forward Managed Services charge is calculated based on the spend of the Provider on which it is enabled unlike the per instance/per hour model. Managed Services are charged on a graduated spend.
+
+Graduated Pricing Tiers:
+
+$0 - $50k = 45%
```
Comment (chrislittle) [inline Release Notes/2018/2018-05-07-cloud-platform-release-notes.md]: probably a table or line breaks needed here. its making it a single line in preview.

File: Release Notes/2018/2018-05-07-cloud-platform-release-notes.md [hunk 2]
```diff
@@ -1,16 +1,54 @@
...
+
+Our initial Managed Services offering included Managed Operating System of Microsoft Windows and RedHat Enterprise Linux flavors. Our expanded set of services now include:
+
+A) Managed Operating System of Microsoft Windows and RedHat Enterprise Linux
+
+B) Managed Applications (Microsoft SQL Server, Microsoft IIS Web Server, Oracle MySQL Enterprise Edition, Apache Web Server, Apache Tomcat)
+
+C) AWS Services (Amazon EC2, VPC, Relational Database Services, EBS, S3, ELB, Route 53, ElastiCache, CloudFront, Management Tools, Cloud Formation, Direct Connect, Cloud Watch, Cloud Trail, Trusted Advisor, Cert Manager, Organizations, Auto Scale, Directory Services, IAM)
+
+D) Azure Services (Virtual Machines, Virtual Network, VPN Gateway, Security Groups, ExpressRoute, IP Addresses, Blob Storage, Queue Storage, Disk Storage
```
Comment (chrislittle) [inline Release Notes/2018/2018-05-07-cloud-platform-release-notes.md]: no closing parenthesis

File: Release Notes/2018/2018-05-07-cloud-platform-release-notes.md [hunk 3]
```diff
@@ -1,16 +1,54 @@
...
+
+###### Supported Platforms
+
+Besides AWS & Azure we are now expanding our supported platforms to include CenturyLink Private Cloud (Dedicated Cloud Compute – Foundation). CenturyLink Private Cloud (DCC-F) users can now signup for Cloud Application Manager and enable Managed Services and get the same features that they would get on AWS & Azure.
+
+###### Improved set of fully supported services:
+
+Our initial Managed Services offering included Managed Operating System of Microsoft Windows and RedHat Enterprise Linux flavors. Our expanded set of services now include:
+
+A) Managed Operating System of Microsoft Windows and RedHat Enterprise Linux
```
Comment (chrislittle) [inline Release Notes/2018/2018-05-07-cloud-platform-release-notes.md]: recommend a formal list using numbering or standard listing in markdown

**Comment by jim-phillips** [comment]
@thomasbroadwell please review @chrislittle's suggestions before you resubmit

**Pull request closed without merging**